### PR TITLE
[Home] add tutorial section

### PR DESCRIPTION
**Pull request by irfandigalih**
image

### CODE REVIEW

File: lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_component.dart [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:timeboxing/Scenes/Page/HomePage/Component/TutorialCard/Model/tutorial_model.dart';
+import 'package:timeboxing/Scenes/Page/HomePage/Component/TutorialCard/tutorial_item.dart';
+import 'package:timeboxing/Shared/Extension/colors_style_extension.dart';
+import 'package:timeboxing/Shared/Extension/text_style_extension.dart';
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_component.dart]: pake package:timeboxing/Shared/Extension/extension_barrel.dart

File: lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_item.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                  height: 75,
+                  decoration: BoxDecoration(
+                    color: TimeBoxingColors.primary50(TimeBoxingColorType.tint),
+                    borderRadius: const BorderRadius.all(Radius.circular(4)),
+                  ),
+                  child: Image.network(
+                      fit: BoxFit.fill, widget.tutorialModel.image),
+                ),
+                const SizedBox(
+                  width: 23,
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_item.dart]: kalo padding margin conventional nya genap lih, kalo nemu ganjil ganti aja ke genap

File: lib/Scenes/Page/HomePage/Model/tutorial_model.dart [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  String text;
+
+  RecommendationCard({
+    required this.id,
+    required this.title,
+    required this.recomendationTittle,
+    required this.imageUrl,
+    required this.text,
+  });
+}
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Model/tutorial_model.dart]: tutorial_model.dart tapi isinya recommendation card?

Comment (irfandigalih) [reply lib/Scenes/Page/HomePage/Model/tutorial_model.dart]: Sorry my brother i will becarefull next time , i just want to make it beautifull add ',' to make it clean

File: lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_component.dart (hunk 1 above)
Comment (irfandigalih) [reply lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_component.dart]: resolve bang gil

File: lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_item.dart (hunk 2 above)
Comment (irfandigalih) [reply lib/Scenes/Page/HomePage/Component/TutorialCard/tutorial_item.dart]: ok bang gil noted

**Review by RenaufalAlrafid — DISMISSED** [review]

**Review by gilangpersada — APPROVED** [review]

**Review by RenaufalAlrafid — APPROVED** [review]

**Pull request merged**